deploy/charts/cert-manager/templates/cainjector-poddisruptionbudget.yaml

```diff
@@ -17,8 +17,8 @@ spec:
       app.kubernetes.io/instance: {{ .Release.Name }}
       app.kubernetes.io/component: "cainjector"
 
-  {{- with .Values.cainjector.podDisruptionBudget.minAvailable }}
-  minAvailable: {{ . }}
+  {{- if and .Values.cainjector.podDisruptionBudget.minAvailable (not .Values.cainjector.podDisruptionBudget.maxUnavailable) }}
```

Comment:
Could you use `hasKey` to make sure the value 0 also works?

Reply:
Right, yaml casting.. Yes I think at least maxUnavailable 0 might make sense if you can only run 1 replica of certmanager for some reason. minAvailable 0 is more or less the same as having no PDB, so for that case I don't think it makes sense, you would just disable the PDB.